cmd/anonymizer/main.go

```diff
@@ -69,9 +70,14 @@ func main() {
 			}
 
 			for _, span := range spans {
-				writer.WriteSpan(&span)
+				if err := w.WriteSpan(&span); err != nil {
+					if errors.Is(err, writer.ErrMaxSpansCountReached) {
+						logger.Error("error while writing span", zap.Error(err))
+						os.Exit(0)
+					}
```

Comment:
```suggestion
					if errors.Is(err, writer.ErrMaxSpansCountReached) {
						os.Exit(0)
					}
					logger.Error("error while writing span", zap.Error(err))
```

Reply:
In this case if ErrMaxSpansCountReached error is encountered, how would the error be logged? As `os.Exit(0)` would cause the program to stop.

Reply:
it's not an error, it does not need to be logged as such. If you want you can add a separate log statement inside if() with Info level saying "max span count processed"

Reply:
Okay, making the changes

Reply:
Made the changes